index.html

```diff
@@ -630,7 +630,43 @@ <h2>DID Documents</h2>
     </section>
 
     <section>
-      <h2>DID Resolvers and DID Resolution</h2>
+      <h2>DID Resolvers</h2>
+
+      <p>
+A <a>DID resolver</a> is a software and/or hardware component that takes a
+<a>DID</a> as input and produces a <a>conforming DID document</a> as output.
+This process is called <a>DID resolution</a> and often includes resolution input
+options and resolution output metadata.
+      </p>
+      <p>
+The general process of <a>DID resolution</a> is described in
+<a href="#did-resolution"></a>. The details of implementing a
+<a>DID resolver</a> are described in the DID Resolution specification
+[[?DID-RESOLUTION]].
+      </p>
+    </section>
+
+    <section>
+      <h2>DID Dereferencers</h2>
+
+      <p>
+A <a>DID URL dereferencer</a> is a software and/or hardware component that takes
+a <a>DID URL</a> as input and produces a resource, which might or might not be a
+<a>conforming DID Document</a> as output. This process is called <a>DID URL
+dereferencing</a> and often includes dereferencing input options and
+dereferencing output metadata. Generally, a <a>DID URL dereferencer</a> uses a
+<a>DID resolver</a> to execute <a>DID resolution</a> on the <a>DID</a> portion
+of a <a>DID URL</a> to obtain a <a>DID document</a>. The <a>DID URL
+dereferencer</a> can then perform additional dereferencing on the <a>DID
+document</a> using the absolute <a>DID URL</a>.
+      </p>
+
+      <p>
+The general process of <a>DID URL dereferencing</a> is described in
+<a href="#did-url-dereferencing"></a>. The details of implementing a <a>DID URL
+dereferencer</a> are described in the DID Resolution specification
+[[?DID-RESOLUTION]].
```

Comment:
```suggestion
The inputs and outputs of the <a>DID URL dereferencing</a> process are defined in
<a href="#did-url-dereferencing"></a>. Additional considerations for implementing a <a>DID URL
dereferencer</a> are discussed in the DID Resolution specification
[[?DID-RESOLUTION]].
```

Reply:
@philarcher I am proposing to change the language back toward what I had in #253, is this a better framing of the connection between the documents for you?

Reply:
That's certainly better, yes, but the word specification is still a problem. [DID-RESOLUTION] is not a specification (I know it is, and IMO the decision not the put it on the rec track is self-harming and irrational but there we are). How about:

The inputs and outputs of the <a>DID URL dereferencing</a> process are defined in
<a href="#did-url-dereferencing"></a>. Additional considerations for implementing a <a>DID URLdereferencer</a>, that are out of scope for this document,  are discussed 
[[?DID-RESOLUTION]].

Could you live with that?

Reply:
How about this, which I think lands a little better and is a bit more concise:

```suggestion
The inputs and outputs of the <a>DID URL dereferencing</a> process are defined in
<a href="#did-url-dereferencing"></a>. Additional considerations for implementing a <a>DID URL
dereferencer</a> are discussed in 
[[?DID-RESOLUTION]].
```

Reply:
Looks like we've met in the middle - thanks Justin - I'm happy with this.